### PR TITLE
Include the Project Document as a Markdown file in the repository and publish it to GitHub Pages

### PR DESCRIPTION
**Pull request by yuiseki**
FIX: #52

## What and Why

- In this pull request, the Project Documnent from the Wiki of the GitHub repository of UN Open GIS DWG-7 was imported into the Git repository as Markdown.
- I also made it possible to publish a website on GitHub Pages based on those Markdown files
  - You can verify that it is working at the following URL
  - https://yuiseki.github.io/7/
  - https://yuiseki.github.io/7/Project_Document/
- These Markdowns were written in GitHub Wiki notation, so the generated website is incomplete, but I believe it is better than nothing
- I believe that a situation where there is garbage that we can all improve upon is better than a situation where there is nothing
- Also, and importantly, I intentionally broke it up into a large number of small documents instead of huge one
  - This is because the strength of a distributed version control system like Git is that many people can work together to improve small files
  - Communicating and maintaining with keep consistency in a huge document can be difficult
  - I do feel that "Small is beautiful"

## Note

- To publish as GitHub Pages, you need to change the Pages setting in the GitHub repository settings

[Image from Gyazo](https://gyazo.com/5b811048569e619164da57de35b5ae13)

### CODE REVIEW

File: .github/workflows/jekyll-gh-pages.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main", "md-project-document"]
```
Comment (yuiseki) [inline .github/workflows/jekyll-gh-pages.yml:7]: This line must be modified before merging as follows

```suggestion
    branches: ["main"]
```

**Review by hfu — APPROVED** [review]
Looks to me a good start. 
Maybe we should re-organized the project document in the next substantive update. We can remove some Wiki-like nature and make our next project document more like a normal document. Thanks!

**Review by smellman — APPROVED** [review]
LGTM

**Review by ubukawa — APPROVED** [review]
Thank you. It looks good to me. Thank you!

**Comment by hfu** [comment]
Many thanks for this update!

**Pull request merged**